docs/billings-mt.md

```diff
@@ -0,0 +1,6 @@
+# Billings
+
+- The best thing about Billings is the view from the top of the Rimrocks, a sandstone bluff that runs along the north side of the Yellowstone valley.
+
+- You will drive up Zimmerman trail to the top of the Rims, and find a spot in one of the parks to pull over.
+- Sunsets are beautiful.
```

Comment:
Correct! I am sure it were some interesting archeological finds when digging for the foundation!